src/components/forms/validator.test.ts

```diff
@@ -128,7 +128,7 @@ describe('Forms > Validators', () => {
   })
 
   describe('mustBeEthereumAddress validator', () => {
-    const MUST_BE_ETH_ADDRESS_ERR_MSG = 'Address should be a valid Ethereum address or ENS name'
+    const MUST_BE_ETH_ADDRESS_ERR_MSG = 'Address should be a valid Ethereum address or blockchain domain'
```

Comment:
I think we should change this to `Address should be a valid Ethereum address, ENS name or Unstoppable domain.` in order to make it as specific as possible what is supported and what isn't

Reply:
Thank you, this is updated